### ISSUE TITLE
design Privacy Policy 

### ISSUE THREAD

**Issue by anitadesigns** [hackforla/expunge-assist#1131]
### Dependency 
- [x] #231 

### Overview
Once Content is done creating the copy for the Privacy Policy, adjust the current wireframes with new copy

### Action Items
- [ ] Ensure headers are consistent with typography 
- [ ] Ensure text color is consistent with the rest of the website 
- [ ] Add Autolayout 
- [ ] Help Content hand over to Dev

### Resources/Notes
If a page of Figma has not been created under WIP, feel free to make one.

**Comment by mariaweissman** [comment]
Closing, as the dependency linked to it (#231) was closed with the following note: "Closing issue since it's outdated; an updated privacy policy will be coming from legal. If any changes need to be made then, we will create a new issue."
